lib/snowplow-emr-etl-runner/emr_job.rb

```diff
@@ -875,6 +1016,19 @@ def output_rdb_loader_logs(region, aws_access_key_id, aws_secret_key, log_level)
                 logger.error l[0]
                 logger.error tmp.read
               end
+
+              # Cannot move an object in S3, so copy and delete
+              # This prevents overwriting the original log file in case this step gets retried.
+              s3.copy_object({
+                bucket: bucket,
+                copy_source: "/#{bucket}/#{key}",
+                key: "#{key}.#{attempt}",
+              })
+              s3.delete_object({
+                bucket: bucket,
+                key: key,
+              })
```

Comment:
This juggling of rdb log files was necessary when I thought the rdb load job could be retried.  However, now I have found out the rdb_load job must not be retried.

I don't know whether to go back and rip out this special handling of rdb log files, or whether to leave it now as potentially useful in future.